### ISSUE TITLE
How to use third party reporters?

### ISSUE THREAD

**Issue by trigoesrodrigo** [mantoni/mochify.js#87]
Is it possible to use a third party reporter with mochify?

I tried with a simple project including 'mocha-jenkins-reporter' and running it through mochify returns the following error:

```
# phantomjs:
"mocha-jenkins-reporter" reporter blew up with error:
Error: failed to require "mocha-jenkins-reporter"
      at null:null
Error: invalid reporter "mocha-jenkins-reporter"
      at null:null
```
I tried both with local modules and global ones without success.

Here is a sample repo with my simple test https://github.com/trigoesrodrigo/mochify-custom-reporter-test

**Comment by mantoni** [comment]
There is no support for custom reporters yet. You're first to ask. I'd accept a pull request, but won't have the time to look into it myself.

Since dynamic requires don't work with Browserify, you would have to build support for this into mocaccino, I guess.

**Comment by mkautzmann** [comment]
+1

**Comment by mantoni** [comment]
This feature was implemented in `mocaccino@1.6.0`. Please reinstall `mochify` and the custom reporters should work. Thanks @mkautzmann!
